github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/kubemacpool/kubemacpool-presubmits-v0.42.yaml

```diff
@@ -0,0 +1,62 @@
+---
+presubmits:
+  k8snetworkplumbingwg/kubemacpool:
+    - name: pull-kubemacpool-unit-test-v0.42
+      context: pull-kubemacpool-unit-test
+      branches:
+        - release-0.42
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      cluster: kubevirt-prow-control-plane
+      max_concurrency: 6
+      labels:
+        preset-podman-in-container-enabled: "true"
+        preset-docker-mirror-proxy: "true"
+        preset-shared-images: "true"
+      spec:
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20230801-94954c0
```

Comment:
do you use latest bootstrap?

once the rehearse finish i will review, thanks

Reply:
I copied from you on https://github.com/kubevirt/project-infra/pull/3042/files#diff-d22ac790b5ca665346d1a474662f8afc4fa327448518d61bf259b6014f4967c8R22

Reply:
seems fine
https://github.com/kubevirt/project-infra/pull/3042#issuecomment-1787236951